src/igvfd/types/file.py

```diff
@@ -1188,7 +1340,8 @@ def assembly(self, request, derived_from):
         }
     )
     def transcriptome_annotation(self, request, derived_from):
-        parent_file_object = request.embed(derived_from[0], '@@object?skip_calculated=true')
+        parent_file_object = request.embed(
+            derived_from[0], '@@object_with_select_calculated_properties?field=transcriptome_annotation')
```

Comment:
probably want to do the same thing for alignment and signal file summaries, add assembly and transcriptome back?

Reply:
they were removed from here: https://github.com/IGVF-DACC/igvfd/pull/1641/files#diff-01e44507c3148abef1d1d7b8ffc39d8f8296c41e1d601cca4fadebf8392de766